src/main/scala/diplomacy/Nodes.scala

```diff
@@ -389,7 +389,7 @@ sealed abstract class MixedNode[DI, UI, EI, BI <: Data, DO, UO, EO, BO <: Data](
   private var bundlesSafeNow = false
   // Accessors to the result of negotiation to be used in LazyModuleImp:
   def out: Seq[(BO, EO)] = {
-    require(bundlesSafeNow, s"${name}.out should only be called from the context of its module implementation")
+//    require(bundlesSafeNow, s"${name}.out should only be called from the context of its module implementation")
```

Comment:
We need to figure out a better way to do this. Let me know if you want to chat about it.